src/backend/marsha/core/tests/api/video/test_initiate_transcript.py

```diff
@@ -0,0 +1,203 @@
+"""Tests for the Video initiate transcript API."""
+
+from datetime import datetime, timezone as baseTimezone
+import json
+from unittest import mock
+
+from django.db.transaction import atomic
+from django.test import TestCase
+
+from marsha.core import defaults, factories, models
+from marsha.core.simple_jwt.factories import (
+    InstructorOrAdminLtiTokenFactory,
+    UserAccessTokenFactory,
+)
+from marsha.websocket.utils import channel_layers_utils
+
+
+class VideoInitiateTranscriptAPITest(TestCase):
+    """Test the "initiate transcript" API of the video object."""
+
+    maxDiff = None
+
+    def test_api_video_initiate_transcript_anonymous_user(self):
+        """Anonymous users should not be able to initiate a transcript."""
+        video = factories.VideoFactory()
+
+        response = self.client.post(
+            f"/api/videos/{video.id}/initiate-transcript/",
+        )
+
+        self.assertEqual(response.status_code, 401)
```

Comment:
Can we use the library `http` to replace the hardcoded value of status response with the `HTTPStatus` object ?